data/mods/Xedra_Evolved/monsters/elementals.json

```diff
@@ -134,6 +134,51 @@
     "special_attacks": [ { "id": "ice_spray" } ],
     "death_drops": [ { "item": "water", "charges": 25, "prob": 100 } ]
   },
+  {
+    "id": "mon_undertow",
+    "type": "MONSTER",
+    "name": { "str": "undertow" },
+    "description": "A pair of frog eyes break the water, with a humanoid shadow below.  The water around it churns against the current.",
+    "default_faction": "undine",
+    "bodytype": "blob",
+    "species": [ "UNDINE" ],
+    "volume": "63000 ml",
+    "weight": "63 kg",
+    "diff": 15,
+    "hp": 70,
+    "speed": 110,
+    "material": [ "fae_flesh" ],
+    "symbol": "U",
+    "color": "cyan",
+    "aggression": 0,
+    "morale": 105,
+    "melee_skill": 3,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "bash", "amount": 2 } ],
+    "dodge": 5,
+    "vision_day": 35,
+    "vision_night": 5,
+    "grab_strength": 40,
+    "anger_triggers": [ "PLAYER_CLOSE" ],
+    "dissect": "dissect_undine_single",
+    "special_attacks": [
```

Comment:
Oooh I like that this is a different effect than the Mavka drowning attempts.

Reply:
There sure are a lot of "spirits who hang out in water and drown people"